device/barefoot/x86_64-accton_wedge100bf_32x-r0/montara/qos.json.j2

```diff
@@ -1,161 +1,19 @@
-{
+{%- macro generate_tc_to_pg_map() %}
     "TC_TO_PRIORITY_GROUP_MAP": {
         "AZURE": {
             "3": "3",
             "4": "4"
         }
     },
-    "MAP_PFC_PRIORITY_TO_QUEUE": {
-        "AZURE": {
-            "0": "0",
-            "1": "1",
-            "3": "3",
-            "4": "4"
-        }
-    },
-    "TC_TO_QUEUE_MAP": {
-        "AZURE": {
-            "0": "0",
-            "1": "1",
-            "3": "3",
-            "4": "4"
-        }
-    },
-    "DSCP_TO_TC_MAP": {
-        "AZURE": {
-            "0":"0",
-            "1":"0",
-            "2":"0",
-            "3":"3",
-            "4":"4",
-            "5":"0",
-            "6":"0",
-            "7":"0",
-            "8":"1",
-            "9":"0",
-            "10":"0",
-            "11":"0",
-            "12":"0",
-            "13":"0",
-            "14":"0",
-            "15":"0",
-            "16":"0",
-            "17":"0",
-            "18":"0",
-            "19":"0",
-            "20":"0",
-            "21":"0",
-            "22":"0",
-            "23":"0",
-            "24":"0",
-            "25":"0",
-            "26":"0",
-            "27":"0",
-            "28":"0",
-            "29":"0",
-            "30":"0",
-            "31":"0",
-            "32":"0",
-            "33":"0",
-            "34":"0",
-            "35":"0",
-            "36":"0",
-            "37":"0",
-            "38":"0",
-            "39":"0",
-            "40":"0",
-            "41":"0",
-            "42":"0",
-            "43":"0",
-            "44":"0",
-            "45":"0",
-            "46":"0",
-            "47":"0",
-            "48":"0",
-            "49":"0",
-            "50":"0",
-            "51":"0",
-            "52":"0",
-            "53":"0",
-            "54":"0",
-            "55":"0",
-            "56":"0",
-            "57":"0",
-            "58":"0",
-            "59":"0",
-            "60":"0",
-            "61":"0",
-            "62":"0",
-            "63":"0"
-        }
-    },
-    "SCHEDULER": {
-        "scheduler.0": {
-            "type":"DWRR",
-            "weight": "25"
-        },
-        "scheduler.1": {
-            "type":"DWRR",
-            "weight": "30"
-        },
-        "scheduler.2": {
-            "type":"DWRR",
-            "weight": "20"
-        }
-    },
+{%- endmacro %}
```

Comment:
Is it possible to define this macro in qos_config.j2?

Reply:
Is this question for @wendani or me?